UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Logging/EventIds.cs

```diff
@@ -550,110 +550,105 @@ public enum EventIds
         /// </summary>
         BessConfigInvalidAttributes = 809108,
 
-        /// <summary>
-        /// 809109 - Bess config duplicate Name attribute
-        /// </summary>
-        BessConfigDuplicateFileCount = 809109,
-
         /// <summary>
         /// 809110 - Bess config validation summary
         /// </summary>
-        BessConfigValidationSummary = 809110,
+        BessConfigValidationSummary = 809109,
```

Comment:
EventId need to be corrected in summary as well for EventIds **809109** to **809128**

Reply:
Resolved